app/views/placements/schools/placements/build/check_your_answers.html.erb

```diff
@@ -30,10 +30,18 @@
 
           <% summary_list.with_row do |row| %>
             <% row.with_key(text: t(".subject")) %>
-            <% row.with_value(text: @placement.subjects.map(&:name).to_sentence) %>
+            <% row.with_value(text: @placement.subject.name) %>
             <% row.with_action(text: t(".change"), href: add_subject_placements_school_placement_build_index_path(@school, :add_subject), visually_hidden_text: t(".add_subject")) %>
           <% end %>
 
+          <% if @placement.subject_has_child_subjects? %>
+            <% summary_list.with_row do |row| %>
+              <% row.with_key(text: t(".additional_subjects")) %>
+              <% row.with_value(text: @placement.additional_subjects.map(&:name).to_sentence) %>
```

Comment:
Could we put this code in the `Placement` model as method? (Assuming we are using this in the decorator as well)